sensitivity/src/main/java/powsybl/tutorials/sensitivity/SensitivityTutorial.java

```diff
@@ -0,0 +1,199 @@
+package powsybl.tutorials.sensitivity;
+
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.computation.local.LocalComputationManager;
+import com.powsybl.contingency.BranchContingency;
+import com.powsybl.contingency.ContingenciesProvider;
+import com.powsybl.contingency.Contingency;
+import com.powsybl.iidm.import_.Importers;
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.Line;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.VariantManagerConstants;
+import com.powsybl.iidm.xml.NetworkXml;
+import com.powsybl.loadflow.LoadFlow;
+import com.powsybl.loadflow.LoadFlowParameters;
+import com.powsybl.sensitivity.*;
+import com.powsybl.sensitivity.converter.JsonSensitivityComputationResultExporter;
+import com.powsybl.sensitivity.converter.SensitivityComputationResultExporter;
+import com.powsybl.sensitivity.factors.BranchFlowPerPSTAngle;
+import com.powsybl.sensitivity.factors.functions.BranchFlow;
+import com.powsybl.sensitivity.factors.variables.PhaseTapChangerAngle;
+import com.rte_france.powsybl.hades2.sensitivity.Hades2SensitivityFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author Agnes Leroy <agnes.leroy at rte-france.com>
+ */
+public final class SensitivityTutorial {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SensitivityTutorial.class);
+    private static final String CONTINGENCY_VARIANT = "contingencyVariant";
+
+    public static void main(String[] args) throws IOException {
+
+        // 1. Import the network from a XML file
+        // The network is described in the iTesla Internal Data Model format.
+        File file = new File(SensitivityTutorial.class.getResource("/sensi_network_12_nodes.xml").getPath());
+        Network network = Importers.loadNetwork(file.toString());
+        // This network consists of 12 buses:
+        //     - we are considering schematic exchanges between France, Belgium, Germany and the Netherlands
+        //     - there are 3 buses per country
+        //     - there is one phase-shift transformer in Belgium
+        //     - there are 4 lines we would wish to monitor in this case:
+        //         - Belgium - France
+        //         - France - Germany
+        //         - Germany - the Netherlands
+        //         - the Netherlands - Belgium
+        //     - we are monitoring the effect of the phase-shift transformer or of the generators' injections on
+        //       each of these lines.
+        // In this tutorial the calculations are done in DC mode, but this can be changed in the config file
```

Comment:
I just commited a change to this comment:
// In this tutorial the power flow is done on active power only, but this can be changed in the config file
// by setting dcMode = false in the hades2-default-parameters. Then, the power flow will handle both P and Q.